Dockerfile.dapper

```diff
@@ -1,16 +1,16 @@
-FROM golang:1.11.4
+FROM golang:buster
```

Comment:
Can you pin this to a specific golang version 1.13.x or 1.14.x?  We like the build to be reproducible at the risk that they become out of date.

Reply:
Sure thing!